api/tslint.json

```diff
@@ -46,7 +46,7 @@
         ],
         "quotemark": false,
         "semicolon": [
-            true,
+            false,
```

Comment:
Don't change this please. I'm not a fan either, but the whole node layer is written with non-semicolon style.